src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
```

Comment:
would be great to add private constructor to this util class so it can't be instanciated from outside